### PR TITLE
[T4A7][W13-B4] Conan Kian

### PR DESCRIPTION
**Pull request by ckjr**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/data/person/Name.java [hunk 1]
```diff
@@ -61,4 +63,48 @@ public int hashCode() {
...
+     * Algorithm from https://en.wikipedia.org/wiki/Levenshtein_distance
+     * @param thisName
+     * @param thatName
+     * @return
+     */
+    private int LevenshteinDistance(String thisName, int thisLength, String thatName, int thatLength) { 
+        int cost;
+
+        /* base case: empty strings */
+        if (thisLength == 0) return thatLength;
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Name.java]: coding standard violation :-1:  you need to use braces even for single line conditionals

**Comment by okkhoy** [comment]
@ckjr 
Looking at your commits, I can see that the order of test-develop is little messed up. Please take a look at the handout and the suggested readings in the assignment description.

Some comments added. Please ack & close the PR after reading comments.

File: src/seedu/addressbook/data/person/Name.java (hunk 1 above)
Comment (ckjr) [reply src/seedu/addressbook/data/person/Name.java:93]: Noted

**Comment by ckjr** [comment]
Acknowledged

**Pull request closed without merging**